src/stories/design-system/IntroductionToCreatingStories.stories.mdx

```diff
@@ -0,0 +1,222 @@
+} as ComponentMeta<typeof BackgroundIcon>;
+```
+
+4. #### In your story file, before the component meta, create a summary of the component as a short description, some context as to why the component was created, as well as its figma link
+```jsx
+const summary = "##Summary\nThe Background icon component may contain various icons to visualize or to give information as an image.\n\n";
+const context = "##Context\nThe Background component was added to create a visual of the information it is trying to portray.\n\n";
+const figmaLink = "##Figma Link\nhttps://www.figma.com/file/yJ8kQwyMxuTZudD90MxgOU/%F0%9F%93%90-Components-and-Patterns?type=design&node-id=103-320&mode=design&t=QFaUGXlsf04huhew-0";
+const markdown = summary + context + figmaLink;
+```
+In the story's component meta, we should put the markdown variable in the description like so:
+```jsx
+export default {
+  title: "Components/BackgroundIcon",
+  component: BackgroundIcon,
+  decorators: [withDesign],
+  parameters: {
+    docs: {
+      description: {
+        component: markdown, 
+      },
+    },
+  },
+} as ComponentMeta<typeof BackgroundIcon>;
+```
+
+5. #### For better documentation, we should put the ```argTypes``` which include properties and their attributes in the component meta as well. Ideally, for each property, it would have a ```description, type, control, and defaultValue```. 
+  - ```description``` is a short message on what the property is about. 
+  - ```type``` is the property's data type. 
+  - ```control``` is the type of control it will show as in the storybook documentation. 
+  - ```defaultValue``` is the value which will be intially loaded in the docs. 
+
+Below is an example of it:
+```jsx
+export default {
+  title: "Components/BackgroundIcon",
+  component: BackgroundIcon,
+  decorators: [withDesign],
+  argTypes: {
+    icon: {
+      description: "Icon of the component from fortawesome",
+    },
+    size: {
+      description: "Size of the component",
+      type: "string",
+      control: {
+        type: "select",
+        options: [
+          "xs",
+          "sm",
+          "lg",
+          "xl",
+        ],
+      },
+      defaultValue: "sm",
+    },
+    cs: {
+      description:
+        "Custom styles of the component. Styles are similar to css styles but in camel case",
+      type: Object,
+    },
+  },
+  parameters: {
+    docs: {
+      description: {
+        component: markdown,
+      },
+    },
+  },
+} as ComponentMeta<typeof BackgroundIcon>;
+``` 
+
+6. #### Next, add a template to be used for your stories:
+More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+```jsx
+const Template: ComponentStory<typeof BackgroundIcon> = (args: any) => (
+  <BackgroundIcon {...args} />
+);
+```
+
+7. #### Then, bind the template and create your first story
+More on args: https://storybook.js.org/docs/react/writing-stories/args
+```jsx
+export const StarXs = Template.bind({});
```

Comment:
Question; in some cases it might be useful to add additional comments / context to individual use cases. Is there a way to describe the example here?

For example, on Avatar, we might explain use of the circle variant for users.